sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -3,479 +3,563 @@
+  // (undocumented)
+  complete(m: Message): Promise<void>;
+  // (undocumented)
+  deadLetter(m: Message): Promise<void>;
+  // (undocumented)
+  defer(m: Message): Promise<void>;
+}
+
 // @public
 export interface CorrelationFilter {
-    contentType?: string;
-    correlationId?: string;
-    label?: string;
-    messageId?: string;
-    replyTo?: string;
-    replyToSessionId?: string;
-    sessionId?: string;
-    to?: string;
-    userProperties?: any;
+  contentType?: string;
+  correlationId?: string;
+  label?: string;
+  messageId?: string;
+  replyTo?: string;
+  replyToSessionId?: string;
+  sessionId?: string;
+  to?: string;
+  userProperties?: any;
 }
 
-export { DataTransformer }
+export { DataTransformer };
 
 // @public
 export interface DeadLetterOptions {
-    deadLetterErrorDescription: string;
-    deadletterReason: string;
+  deadLetterErrorDescription: string;
+  deadletterReason: string;
 }
 
-export { delay }
+export { delay };
 
-export { Delivery }
+export { Delivery };
 
 // @public
-export type EntityStatus = "Active" | "Creating" | "Deleting" | "ReceiveDisabled" | "SendDisabled" | "Disabled" | "Renaming" | "Restoring" | "Unknown";
+export type EntityStatus =
+  | "Active"
+  | "Creating"
+  | "Deleting"
+  | "ReceiveDisabled"
+  | "SendDisabled"
+  | "Disabled"
+  | "Renaming"
+  | "Restoring"
+  | "Unknown";
+
+export { HttpOperationResponse };
 
-export { HttpOperationResponse }
+// @public (undocumented)
+export type Message = Omit<
+  ServiceBusMessage,
+  "complete" | "abandon" | "defer" | "deadletter" | "prototype"
+>;
 
 // @public
 export type MessageCountDetails = {
-    activeMessageCount: number;
-    deadLetterMessageCount: number;
-    scheduledMessageCount: number;
-    transferMessageCount: number;
-    transferDeadLetterMessageCount: number;
+  activeMessageCount: number;
+  deadLetterMessageCount: number;
+  scheduledMessageCount: number;
+  transferMessageCount: number;
+  transferDeadLetterMessageCount: number;
 };
 
 // @public
 export interface MessageHandlerOptions {
-    autoComplete?: boolean;
-    maxConcurrentCalls?: number;
-    maxMessageAutoRenewLockDurationInSeconds?: number;
+  autoComplete?: boolean;
+  maxConcurrentCalls?: number;
+  maxMessageAutoRenewLockDurationInSeconds?: number;
 }
 
-export { MessagingError }
+export { MessagingError };
+
+// @public
+export interface NonSessionReceiverTrack2<LockModeT extends "peekLock" | "receiveAndDelete"> {
+  // Warning: (ae-forgotten-export) The symbol "MessageIterator" needs to be exported by the entry point index.d.ts
+  //
+  // (undocumented)
+  iterateMessages(): MessageIterator<ContextType<LockModeT>>;
+  // Warning: (ae-forgotten-export) The symbol "MessageHandlers" needs to be exported by the entry point index.d.ts
+  // Warning: (ae-forgotten-export) The symbol "ContextType" needs to be exported by the entry point index.d.ts
+  //
+  // (undocumented)
+  streamMessages(handlers: MessageHandlers<ContextType<LockModeT>>): void;
+}
```

Comment:
`receiveBatch(maxMessages, cancellationToken)` is missing.

Reply:
Added.